ILRepack.IntegrationTests/packages.config

```diff
@@ -1,4 +1,13 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
+  <package id="BamlParser" version="1.0.0" targetFramework="net40" />
```

Comment:
Only Rx & Zip are required to DL NuGet packages, the rest is because this project now executes ILRepack (and needs its dependencies). So you're saying to use NuGet.Core instead of WebClient + Unzip (I guess).
Why not, I'm not familiar with the lib, I'd have to check whether its use makes things easier.

Thx